src/Plugins/RabbitMQ/Subscriber/RabbitMqMessageSubscriberService.cs

```diff
@@ -438,5 +394,32 @@ private static MessageReceivedEventArgs CreateMessage(string topic, BasicDeliver
                 deliveryTag: eventArgs.DeliveryTag.ToString(CultureInfo.InvariantCulture)),
                 CancellationToken.None);
         }
+        private bool QueueExists(string queueName, ChannelType type, string hostName, string username, string password, string virtualHost, string useSSL, string portNumber)
+        {
+            var testChannel = _rabbitMqConnectionFactory.MakeTempChannel(ChannelType.Subscriber, _endpoint, _username, _password, _virtualHost, _useSSL, _portNumber);
+
+            try
+            {
+                var testRun = testChannel!.QueueDeclarePassive(queue: queueName);
+            }
+            catch (OperationInterruptedException operationInterruptedException)
+            {
+                ///RabbitMQ node that hosts the previously created dead-letter queue is unavailable
+                if (operationInterruptedException.Message.Contains("down or inaccessible"))
+                {
+                    _logger.DetectedInaccessibleNodeThatHousesDeadLetterQueue(queueName);
+                }
+                else
+                {
+                    return false;
+                }
+            }
+            catch (Exception ex)
+            {
+                var me = ex.Message;
```

Comment:
If you just catch the exception without naming the exception, you should be able to drop this line.

Reply:
Were there other types of exceptions that we wanted to catch here though?

Reply:
just had a quick look and the exception you get when the queue doesn't exist is still a OperationInterruptedException, so I have removed the general catch now.